### DISCUSSION TITLE
Commits not updating on github contribution calendar

### DISCUSSION THREAD

**Discussion by aruadecarvalho** [community/community#41682] [Repositories]
### Body

I made I think three commits yesterday and one today and all of them didn't register on my calendar, why is this happening?

**Comment by radar-blip** [comment]
Its showing two for yesterday, and one for today. 👍🏻  Maybe a time-zone thing.

![comms](https://user-images.githubusercontent.com/39196767/207499226-cf408582-a279-47a3-9bb3-1733f34d1fac.png)

**Reply by aruadecarvalho** [reply]
probably, because for me it doesnt show
![image](https://user-images.githubusercontent.com/103606795/207615261-b0542e6b-b207-47ae-91bc-7fe1c813e3f6.png)
